src/lib/db/client-metrics-store-v2.ts

```diff
@@ -95,6 +95,39 @@ const variantRowReducer = (acc, tokenRow) => {
     return acc;
 };
 
+const variantRowReducerV2 = (acc, tokenRow) => {
+    const {
+        feature_name: featureName,
+        app_name: appName,
+        environment,
+        timestamp,
+        date,
+        yes,
+        no,
+        variant,
+        count,
+    } = tokenRow;
+    const key = `${featureName}_${appName}_${environment}_${
+        timestamp || date
+    }_${yes}_${no}`;
+    if (!acc[key]) {
+        acc[key] = {
+            featureName,
+            appName,
+            environment,
+            timestamp: timestamp || endOfDay(date),
```

Comment:
this is the new code: timestamp for days will be taken from the DB date and moved to the end of the day